src/Raven.Server/Documents/Handlers/StatsHandler.cs

```diff
@@ -67,7 +67,7 @@ private void FillDatabaseStatistics(DatabaseStatistics stats, DocumentsOperation
                 stats.CountOfConflicts = Database.DocumentsStorage.ConflictsStorage.ConflictsCount;
                 stats.SizeOnDisk = size.Data;
                 stats.NumberOfTransactionMergerQueueOperations = Database.TxMerger.NumberOfQueuedOperations;
-                stats.CountOfCounters = Database.DocumentsStorage.CountersStorage.GetNumberOfCounterEntries(context);
+                stats.CountOfCounters = Database.DocumentsStorage.CountersStorage.GetNumberOfCounterEntries();
```

Comment:
Remove `CountOfCounters`

Reply:
@ayende can we have the number of "counter entries" here (instead of the percise counters count) this way we'll have an indication of the order of magnitude of counters. It's the same as tracking the attachment count - there could be 1kb attachment and 10mb attachment, we still treat the equally in it's count field

Reply:
Fine by me, yes, but needs to be carefully explained.